packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/ToolbarActionWidget.tsx

```diff
@@ -350,12 +347,16 @@ export const ToolbarActionWidget = ({
       <div
         data-testid={`ToolbarAction-DropArea-${toolbarAction.id}`}
         className={classes.placeholder}
-        onDragEnter={handleDragEnter}
-        onDragOver={handleDragOver}
-        onDragLeave={handleDragLeave}
-        onDrop={handleDrop}
+        onDragEnter={readOnly ? noop : handleDragEnter}
```

Comment:
May be it would have been better/more readable to modify handleDragEnter/handleDragOver/handleDragLeave... instead.
What do you think about it?